### PR TITLE
Cancel previous xhr request on server side pagination

### PR DESCRIPTION
**Pull request by D0d0**
Cancel previous xhr request to server.

### CODE REVIEW

**Comment by wenzhixin** [comment]
can you provide an example?

**Comment by 719media** [comment]
+1, i am not sure if this needs an example. just have an ajax request take 10+ seconds on a forced sleep or something and then notice them pile up as you paginate.

File: src/bootstrap-table.js [hunk 1]
```diff
@@ -1846,7 +1846,10 @@
         if (this.options.ajax) {
             calculateObjectValue(this, this.options.ajax, [request], null);
         } else {
-            $.ajax(request);
+            if(this._xhr) {
```
Comment (wenzhixin) [inline src/bootstrap-table.js]: `if (this._xhr && this._xhr.readyState !== 4) {`

**Comment by wenzhixin** [comment]
@D0d0

**Pull request merged**